diff --git a/src/gameoflife.py b/src/gameoflife.py
index 2aab2d2..59e4ae5 100644
--- a/src/gameoflife.py
+++ b/src/gameoflife.py
@@ -46,7 +46,7 @@ def getGrid(grid, N):
         livecount += 1
     return newGrid
 
-def update(frameNum, img, N, grid, ):
+def update(frameNum, img, N, grid, totalFrames):
     #plt.pause(1)
     # Copy grid to generate the image to print vs. the data packed cell world
     newImg = np.zeros(shape=(N,N,3))
